### PR TITLE
Improvement/update demo react

### PR DESCRIPTION
**Pull request by iths-jonathan-engman**
**Describe pull-request**  
A fresh installation of React demo

**Solving issue**  
Updated React demo with fresh installation

**How to test**  
-- Pull down
-- Initiate the project

**Issues**
-- On line 38 in components/main/Main.js, the "a" element doesn't href to anything, and React doesnt like that. If a dummy link is set with href=, like "/" or "/#", the disabled breadcrumb doesn't get "disabled", and becomes clickable.

**Screenshots**  
image
image

### CODE REVIEW

File: demo/react/src/App.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import Header from './components/header/Header';
```
Comment (mmexvr) [inline demo/react/src/App.js:1]: Match the exact name with uppercase or not :)

**Review by mmexvr — CHANGES_REQUESTED** [review]

**Review by mmexvr — APPROVED** [review]

**Comment by helloimela** [comment]
Same here, make sure package-lock.json contains the latest package name and no changes occured when you run npm i once again. Check also in Vue PR #118

File: demo/react/public/manifest.json [hunk 2]
```diff
@@ -6,6 +6,16 @@
       "src": "favicon.ico",
       "sizes": "64x64 32x32 24x24 16x16",
       "type": "image/x-icon"
+    },
+    {
+      "src": "logo192.png",
```
Comment (helloimela) [inline demo/react/public/manifest.json:11]: Is this file exist somewhere? What is this image used for?
I got this error on the browser 
image

File: demo/react/public/manifest.json [hunk 3]
```diff
@@ -6,6 +6,16 @@
...
+      "type": "image/png",
+      "sizes": "192x192"
+    },
+    {
+      "src": "logo512.png",
```
Comment (helloimela) [inline demo/react/public/manifest.json:16]: same with this one

**Review by helloimela — APPROVED** [review]

**Pull request merged**